hw1.html

```diff
@@ -0,0 +1,39 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <title>Cubes</title>
+  </head>
+  <body>
+    <div class="row" id=widget>
+        <div class="col" id="Chicago"><img src="chicago.png" class="rounded mx-auto d-block" alt="rain"> </br> Chicago / 23°</div>
```

Comment:
Layout grid should be separated from the components. It would be better to use bootstrap classes as wrappers around your actual components. https://getbootstrap.com/docs/4.0/examples/grid/
Components should be created using the bem naming : https://ru.bem.info/ 
ex.: This is the layout grid
![2](https://user-images.githubusercontent.com/53054365/94272790-3a5d1e80-ff4c-11ea-9969-85263f43453b.jpg)

Now, separatly from this layout grid we create our component's structure using BEM naming. All our custom css styles, should be applied to these custom classes:
![1](https://user-images.githubusercontent.com/53054365/94272722-21ed0400-ff4c-11ea-8afe-e4df770b091b.jpg)